overlord/devicestate/firstboot_test.go

```diff
@@ -445,9 +442,18 @@ func (s *FirstBootTestSuite) TestPopulateFromSeedHappy(c *C) {
 func (s *FirstBootTestSuite) TestPopulateFromSeedMissingBootloader(c *C) {
 	chg := s.makeBecomeOpertionalChange(c)
 	st := s.overlord.State()
+
+	// run the change, we cannot use s.overlord.Settle() here as each
+	// "DeviceManager.Ensure()" will call ensureSeedYaml which will
+	// create a new "seed" change so things never settle down
+	s.overlord.Loop()
+	defer s.overlord.Stop()
+	for !chg.IsReady() {
+		// nothing
+	}
+
```

Comment:
ok, I'll try to play with this locally myself tomorrow, it might be that the test was problematic already and some change here triggered something.